spec/helpers.lua

```diff
@@ -6,7 +6,7 @@
 -- @module spec.helpers
 
 local BIN_PATH = "bin/kong"
-local TEST_CONF_PATH = "spec/kong_tests.conf"
+local TEST_CONF_PATH = os.getenv("TEST_CONF_PATH") or "spec/kong_tests.conf"
```

Comment:
```suggestion
local TEST_CONF_PATH = os.getenv("KONG_SPEC_TEST_CONF_PATH") or "spec/kong_tests.conf"
```

Please prefix any test-only vars with `KONG_SPEC_`